src/plugins/cosmos/components/modals/Staking/views/Overview.tsx

```diff
@@ -107,25 +109,35 @@ export const Overview: React.FC<StakedProps> = ({
                   .decimalPlaces(asset.precision)}
                 fiatRate={bnOrZero(marketData.price)}
                 renderButton={() => (
-                  <ClaimButton assetId={assetId} validatorAddress={validatorAddress} />
+                  <ClaimButton
+                    assetId={assetId}
+                    validatorAddress={validatorAddress}
+                    isDisabled={bnOrZero(rewardsAmount).isZero()}
```

Comment:
Is it ever exactly zero? From testing, it looks like even after you claim, you will quickly have a very small balance again (to 6+ decimal places of 0.$), I reckon at new blocktime. So I don't think this will disable the button the way we think it does.

Reply:
As we talked about in private, this is only in case you don't have any active staking, because you have 0 rewards exactly. If you want to disable the button when your rewards are really low, it needs product input in order to decide but It makes no sense to me.